Source/Core/Core/PrimeHack/Mods/FpsControls.h

```diff
@@ -18,7 +18,17 @@ class FpsControls : public PrimeMod {
   void run_mod(Game game, Region region) override;
   void init_mod(Game game, Region region) override;
 
+
+
 private:
+  // Beam/Visor Menu Control Enum:
```

Comment:
I'm being very pedantic, but the comment isn't needed, the enum name states what it is. :P